tests/integration/v2/app.go

```
@@ -397,6 +427,31 @@ func (a *App) SignCheckDeliver(
 	return txResult
 }
 
+// RunMsg runs the handler for a transaction message.
+// It required the context to have the integration context.
+// a new state is committed if the option WithAutomaticCommit is set in options.
+func (app *App) RunMsg(t *testing.T, ctx context.Context, handler handler, option ...Option) (resp transaction.Msg, err error) {
```

golangci-lint (1.62.2)

test helper function should start from t.Helper()

(thelper)